.github/workflows/azure-e2e.yaml

```diff
@@ -0,0 +1,52 @@
+'on':
+  pull_request:
+    branches:
+    - '*'
+permissions:
+  contents: read
+  id-token: write
+name: E2E  Tests - Build Azure Image
+jobs:
+  rune2e:
+    strategy:
+      fail-fast: false
+      max-parallel: 10
+      matrix:
+        os:
+        - "ubuntu 20.04"
+        buildConfig:
+        - "basic"
+    runs-on: self-hosted
+    continue-on-error: true
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: ${{ github.ref }}
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version-file: 'go.mod'
+
+      - name: Docker Login
+        uses: docker/login-action@v2
+        with:
+          registry: docker.io
+          username: mesosphereci
```

Comment:
can we use org level secrets. `DOCKER_READ_WRITE_USERNAME` and `DOCKER_READ_WRITE_PASSWORD` ?
Assuming that they should be ok to use even it for a public repo.

Reply:
Those are not org level secrets

Reply:
Used other secrets